doc/index.md

```diff
@@ -1,56 +1,59 @@
 # Getting Started
 
-Npgsql has an Entity Framework Core provider. It mostly behaves like a any other EFCore provider (e.g. SQL Server) - all the information in the [general EF Core docs](https://docs.microsoft.com/en-us/ef/core/index) applies. If you're just getting started with EF Core, those docs are the best place to start.
+Npgsql has an Entity Framework (EF) Core provider. It behaves like other EF Core provider (e.g. SQL Server), so all of the information in the [general EF Core docs](https://docs.microsoft.com/en-us/ef/core/index) applies. If you're just getting started with EF Core, those docs are the best place to start.
 
 Development happens in the [Npgsql.EntityFrameworkCore.PostgreSQL](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL) repository, all issues should be reported there.
 
-## Using the Npgsql EF Core Provider
+## Configuring the project file
 
-To use the Npgsql EF Core provider, simply add a dependency on `Npgsql.EntityFrameworkCore.PostgreSQL`. You can follow the instructions in the general [EF Core Getting Started docs](https://docs.microsoft.com/en-us/ef/core/get-started/).
+To use the Npgsql EF Core provider, add a dependencies for `Npgsql` and `Npgsql.EntityFrameworkCore.PostgreSQL`. You can follow the instructions in the general [EF Core Getting Started docs](https://docs.microsoft.com/en-us/ef/core/get-started/).
```

Comment:
I don't think a dependency needs to be added for Npgsql, a dependency on Npgsql.EntityFrameworkCore.PostgreSQL should be enough? Also "add a dependencies" is wrong.